src/handlers/issue/generate-permits.ts

```diff
@@ -47,17 +51,44 @@ async function generateComment(totals: TotalsById, issue: GitHubIssue, config: B
 
     const beneficiaryAddress = data.length > 0 ? data[0].wallets.address : "";
 
+    const permits = [];
     const permit = await generatePermit2Signature({
       beneficiary: beneficiaryAddress,
       amount: tokenAmount,
       userId: userId,
+      organizationName: issue.repository_url.split("/").slice(-2)[0],
+      repositoryName: issue.repository_url.split("/").slice(-1)[0],
+      issueNumber: issue.number.toString(),
       config,
     });
-
     permits.push(permit);
+    allTxs.push(permit);
+
+    const nftMints = [];
+    if (isNftRewardEnabled && userTotals.details.length > 0) {
+      const contributions = userTotals.details.map((detail) => detail.contribution).join(",");
+      const nftMint = await generateNftMintSignature({
+        organizationName: issue.repository_url.split("/").slice(-2)[0],
+        repositoryName: issue.repository_url.split("/").slice(-1)[0],
+        issueNumber: issue.number.toString(),
+        beneficiary: beneficiaryAddress,
+        username: contributorName,
+        contributionType: contributions,
+      });
+      nftMints.push(nftMint);
+      allTxs.push(nftMint);
+    }
+
+    const claimData = [
+      ...permits.map((permit) => ({ type: "permit", ...permit })),
+      ...nftMints.map((nftMint) => ({ type: "nft-mint", ...nftMint })),
```

Comment:
Perhaps it makes sense to be more deliberate with the variable naming. For example, both are permits. One is for fungible tokens, and the other for NFTs. Examples:

- `erc20CompliantPermit`
- `erc721CompliantPermit`

Or

- `nonFungiblePermit`
- `fungiblePermit`

Reply:
makes sense!